### ISSUE TITLE
Provide builds for arm64

### ISSUE THREAD

**Issue by hollow** [jmatsu/terraform-provider-slack#57]
Please provide builds for `{darwin,linux}_arm64` so this provider can be used out-of-the-box on M1 based machines or similar.

```
│ Error: Incompatible provider version
│ 
│ Provider registry.terraform.io/jmatsu/slack v0.8.0 does not have a package available for your current platform, darwin_arm64.
```

Thank you!

**Comment by jmatsu** [comment]
Sorry for overlooking this issue. I guess it would be possible but please give me some time. PRs are welcome by the way.

**Comment by jmatsu** [comment]
I could build the binary targeting darwin-arm64, however, I don't have any M1 machine. Can any of you try darwin-arm64 binary?

https://github.com/jmatsu/terraform-provider-slack/actions/runs/2536917292

**Comment by jmatsu** [comment]
darwin-arm64 binary is available since v0.8.1. Please check it out and open an issue if you have any trouble.

![ss 2022-06-25 at 13 14 01](https://user-images.githubusercontent.com/4340693/175757651-bb86c05e-19cb-4fd7-ae14-15fc4ddd0cdf.png)
